src/App.css

```diff
@@ -1,39 +1,11 @@
-.App {
```

Comment:
![branch](https://user-images.githubusercontent.com/93039359/188733220-12c97145-6e4d-4e9c-a081-b22022cf7d94.png)

- [ ] Please follow Gitflow because you should have made this branch from your dev branch instead of the main branch.
- [ ] Your dev branch should be your default branch :+1: